vuegridlayout.css

```diff
@@ -23,6 +23,11 @@ div#layoutApp .layoutJSON {
   margin: 0;
 }
 
+.layoutJSON button.current-tableau-button {
+  background: yellow !important;
+  padding: 2px 10px;
```

Comment:
Not sure if this is what you intended, but changing the padding from the default `.btn-default` creates an interesting effect of the focused button *popping out*, but it leads to the button row jiggling as the larger button is accommodated.

You can dodge the jiggling and still keep the popout effect by trading `margin` for `padding` on the focused button. For example, if `index.html` has:

```html
                class="btn btn-default btn-xs tableau-button"
                v-bind:class="{ 'current': isCurrentTableau(t) }"
```

and you have CSS:

```css
.layoutJSON button.tableau-button {
  margin: 5px 10px;
  padding: 2px 5px;
}

.layoutJSON button.tableau-button.current {
  background: yellow;
  margin: 2px 5px;
  padding: 5px 10px;
}
```

Reply:
That's cool! Didn't know you could do that.